ui/apps/platform/cypress/helpers/sort.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Assert that each pair of adjacent items are in sorted order.
+ * items might be DOM elements from cy.get(…) call.
+ * callbackForPairOfSortedItems(itemA, itemB) is returned from higher-order function below.
+ */
+export function assertSortedItems(items, callbackForPairOfSortedItems) {
+    for (let indexB = 1; indexB < items.length; indexB += 1) {
+        const itemA = items[indexB - 1];
+        const itemB = items[indexB];
+        const result = callbackForPairOfSortedItems(itemA, itemB);
+
+        // No news is good news. One incorrect result is enough bad news.
+        if (result !== '') {
+            expect(result).to.equal('');
+            break;
+        }
+    }
+}
+
+/*
+ * Create callback function for assertSortedItems function above.
+ * notValidDescription looks like a function name
+ * - Its prefix is not.
+ * - Its root is a description in PascalCase of valid and sorted values from items.
+ * getValueFromItem encapsulates access like item?.innerText property.
+ * isValidValue encapsulates expected type of value like string.
+ * isPairOfSortedValues encapsulates comparison like ascending or descending.
+ *
+ * Return empty string for a pair of valid and sorted values.
+ * Return non-empty string which looks like a function call for a pair of invalid or unsorted values.
+ */
+export function createCallbackForPairOfSortedItems(
+    notValidDescription,
+    getValueFromItem,
+    isValidValue,
+    isPairOfSortedValues
+) {
+    return function callbackForPairOfItems(itemA, itemB) {
+        const valueA = getValueFromItem(itemA);
+        const valueB = getValueFromItem(itemB);
+
+        if (isValidValue(valueA) && isValidValue(valueB) && isPairOfSortedValues(valueA, valueB)) {
+            return ''; // return empty string for correctly sorted pair
```

Comment:
I like the way this file is setup. Returning the empty string felt a little odd at first but it makes sense now that I understand whats going on.

Reply:
The return value was an unexpected change in direction.
The first draft imitated `compareFn` for array sort method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description